auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/request_handlers/api/LoginApiRequestHandler.kt

```diff
@@ -44,6 +44,7 @@ class LoginApiRequestHandler : ApiRequestHandler {
         if (args.getOrDefault("parameters", null) is HashMap<*, *>) {
             loginBuilder.addParameters(args["parameters"] as Map<String, String>)
         }
+        loginBuilder.validateClaims();
```

Comment:
I don't think we should enable this without also providing a way to customize the ID Token validation (e.g. the issuer -> `withIdTokenVerificationIssuer ()` in Auth0.Android). Since this would be outside the scope of this PR, better leave this for another PR.

Reply:
I removed these